packages/graphql/src/services/learningElements.ts

```diff
@@ -219,9 +219,14 @@ export async function respondToQuestionInstance(
       }
 
       case QuestionType.NUMERICAL: {
-        if (!response.value) {
-          break
+        if (
+          !response.value ||
```

Comment:
```suggestion
          typeof response.value === 'undefined' ||
```

Reply:
Shouldn't be an issue for strings, but I still adjusted it to test for undefined values, empty strings and null values as well (just to be sure 😄)